tasklauncher/spring-cloud-starter-stream-sink-tasklauncher-cloudfoundry/src/main/resources/META-INF/spring-configuration-metadata-whitelist.properties

```diff
@@ -0,0 +1 @@
+configuration-properties.classes=
```

Comment:
2) Indeed there is a problem here. Two problems even, which come from boot limitations I think: a) both sets of properties appear and b) the ones using the shorter prefix have no documentation, because source of said classes is not available at time of build (we've been bitten by this issue before).

We can work around a) by citing whitelists *[by-name](http://docs.spring.io/spring-cloud-dataflow/docs/1.1.0.BUILD-SNAPSHOT/reference/html/spring-cloud-dataflow-register-apps.html#spring-cloud-dataflow-stream-app-whitelisting)* instead of *by-class*.